types/nightwatch/index.d.ts

```diff
@@ -1,11 +1,12 @@
-// Type definitions for nightwatch 0.9
+// Type definitions for nightwatch 1.0
```

Comment:
Are these changes to support nightwatch@1.0? From the issue comment thread, that does not seem to be the case.